### PR TITLE
Add cmd make test-e2e-sts to e2e-aws-manual-oidc

### PR DESCRIPTION
**Pull request by bentito**
For Cloud Credential Operator.

Adds a test `make test-e2e-sts` to the already established workflow: `openshift-e2e-aws-manual-oidc-sts`

### CODE REVIEW

**Comment by bentito** [comment]
/pj-rehearse pull-ci-openshift-cloud-credential-operator-master-e2e-aws-manual-oidc

**Comment by bentito** [comment]
/pj-rehearse pull-ci-openshift-cloud-credential-operator-master-e2e-aws-manual-oidc

**Comment by bentito** [comment]
/pj-rehearse pull-ci-openshift-cloud-credential-operator-master-e2e-aws-manual-oidc

**Comment by bentito** [comment]
/pj-rehearse pull-ci-openshift-cloud-credential-operator-master-e2e-aws-manual-oidc

**Comment by bentito** [comment]
/pj-rehearse pull-ci-openshift-cloud-credential-operator-master-e2e-aws-manual-oidc

**Comment by bentito** [comment]
/pj-rehearse pull-ci-openshift-cloud-credential-operator-master-e2e-aws-manual-oidc

**Comment by bentito** [comment]
/pj-rehearse pull-ci-openshift-cloud-credential-operator-master-e2e-aws-manual-oidc

**Comment by bentito** [comment]
/pj-rehearse pull-ci-openshift-cloud-credential-operator-master-e2e-aws-manual-oidc

File: ci-operator/config/openshift/cloud-credential-operator/openshift-cloud-credential-operator-master.yaml [hunk 1]
```diff
@@ -100,6 +100,14 @@ tests:
   optional: true
   steps:
     cluster_profile: aws
+    test:
+    - as: e2e-test-sts
+      commands: make test-e2e-sts
+      from: builder
```
Comment (stevekuznetsov) [inline ci-operator/config/openshift/cloud-credential-operator/openshift-cloud-credential-operator-master.yaml:106]: The error you're seeing is that there is nothing called `builder` in the system - if you meant to reference this base image:

```yaml
  ocp_builder_rhel-8-golang-1.20-openshift-4.14:
    name: builder
    namespace: ocp
    tag: rhel-8-golang-1.20-openshift-4.14
```

I think you need to use the qualified name: `ocp_builder_rhel-8-golang-1.20-openshift-4.14`

Comment (bentito) [reply ci-operator/config/openshift/cloud-credential-operator/openshift-cloud-credential-operator-master.yaml:106]: so complete as in:
```yaml
...
  from: ocp_builder_rhel-8-golang-1.20-openshift-4.14
...
```
and then that would also have the src context as well, that is, both make and the Makefile would be there?

**Comment by bentito** [comment]
/pj-rehearse pull-ci-openshift-cloud-credential-operator-master-e2e-aws-manual-oidc

**Comment by bentito** [comment]
/pj-rehearse pull-ci-openshift-cloud-credential-operator-master-e2e-aws-manual-oidc

**Comment by bentito** [comment]
/pj-rehearse pull-ci-openshift-cloud-credential-operator-master-e2e-aws-manual-oidc

**Comment by bentito** [comment]
/pj-rehearse pull-ci-openshift-cloud-credential-operator-master-e2e-aws-manual-oidc

**Comment by bentito** [comment]
/pj-rehearse pull-ci-openshift-cloud-credential-operator-master-e2e-aws-manual-oidc

**Comment by bentito** [comment]
Ok after awhile of different CI problems, that rehearse now shows it properly calling the AWS STS e2e stub that was added to CCO's make targets. This can get LGTM and APPROVED please @abutcher or @suhanime

**Comment by bentito** [comment]
> @bentito: you cannot LGTM your own PR.

But you can try

**Pull request merged**